src/domain/session/rightpanel/comparator.js

```diff
@@ -0,0 +1,97 @@
+import {PowerLevels} from "../../../matrix/room/timeline/PowerLevels.js";
+
+export function createMemberComparator(powerLevels) {
+    const collator = new Intl.Collator();
+    const removeCharacter = string => string.charAt(0) === "@"? string.slice(1) : string;
```

Comment:
Perhaps removing the explaining comment there just hours ago was a mistake :smile:
This is so that we can spread the members whose name is just their user id (starting with @) from clubbing together at the top of the list. By ignoring the starting @, these members are alphabetically sorted on their user id.